ambari-server/src/main/java/org/apache/ambari/server/orm/entities/TopologyRequestEntity.java

```diff
@@ -75,6 +75,10 @@
   @Column(name = "description", length = 1024, nullable = false)
   private String description;
 
+  @Lob
+  @Column(name = "raw_request_body", length = 30000, nullable = false)
+  private String rawRequestBody;
```

Comment:
On large deployments, could this value be too small?

Reply:
What do you think should be a good value?